internal/xds/types/resourceversiontable_test.go

```diff
@@ -233,7 +278,7 @@ func TestAddOrReplaceXdsResource(t *testing.T) {
 			name: "inject-new-listener",
 			tableIn: &ResourceVersionTable{
 				XdsResources: XdsResources{
```

Comment:
was this an existing bug ?

Reply:
Thanks for the review.
I believe the code was carried over from the main branch when I created my branch.
I cannot recall making any changes to it.

But I agree that 'resourcev3.ListenerType: []types.Resource{},' makes more sense. We need to test adding a new listener to the table, so we wouldn't need to initialize 'tableIn' with the 'testListener'.